### ISSUE TITLE
device and area registry support

### ISSUE THREAD

**Issue by maxandersen** [home-assistant/home-assistant-cli#151]
Would be nice to list/query devices and manipulate areas.

```
$ hass-cli area list
AREA
Bathroom
Winter Garden
Kitchen
Guest Room

$ hass-cli area create "Small Bathroom"

$ hass-cli area delete "Small Bathroom"

$ hass-cli device list "Kitchen.*"

# make inventory overview
$ hass-cli --no-headers --columns manufacturer,model device list | sort | uniq -c | sort -nr 
  10 IKEA of Sweden      TRADFRI bulb GU10 WS 400lm
   8 Philips             SML001
   7 LUMI                lumi.sensor_magnet.aq2
   6 Sonos               Play:1
   6 Philips             LTW012
   6 Philips             LCT003
   6 Philips             Hue color spot
   6 Philips             Hue ambiance candle
   5 Philips             LWG001

# Assign area "Kitchen" to any entity matching "Kitchen Light" filter.
$ hass-cli device assign --filter Kitchen "Kitchen Light"
```

**Comment by maxandersen** [comment]
depends on #39 as the registries are only in websocket api

**Comment by maxandersen** [comment]
fixed in #152
